frontend/packages/console-demo-plugin/src/plugin.tsx

```diff
@@ -277,6 +280,21 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: 'TEST_MODEL_FLAG',
     },
   },
+  {
+    type: 'HorizontalNavTab',
```

Comment:
**NOTE:** It appears that the `@console/demo-plugin` is broken (circular dependency issue). But I've tested this plugin using the planned feature addition of https://github.com/openshift/console/pull/3874.

Reply:
Thanks for the heads-up, I will try to look into it.

Regardless, thanks @alecmerdler for keeping the demo plugin in mind when adding new extension type.